src/main/java/com/app/service/graph/Graph.java

```diff
@@ -63,26 +72,58 @@ private static JFreeChart createChart(String title, String yaxisName1, String ya
         chart.setBorderVisible(false);
         chart.removeLegend();
 
+        if (running) {
+            series1.start();
+            series2.start();
+            return chart ;
+        }
+        if (!running) {
+            if (data != null) {
+                parseAndAddData(data);
+                return chart ;
+            }
+        }
+        chart = null;
+        return null;
+    }
 
-        series1.start();
-        series2.start();
+    public static void  parseAndAddData(File data) throws FileNotFoundException {
+        Scanner scanner = new Scanner(data);
+        ArrayList<ArrayList<Double>> all_values = new ArrayList<ArrayList<Double>>();
+        while(scanner.hasNext()){
+            try {
+                ArrayList<Double> values_long = inputChange(scanner.nextLine());
+                all_values.add(values_long);
+            } catch (Exception e) {
+            }
```

Comment:
tuto to treba vyriesit, nenechavaj to prazdne, ak sa ti s tym nechce teraz parat, nahod tam '// TODO: {comment}' a vytvor si na to issue, aby sa na to nezabudlo -> ak bude issue, moze sa na to pozriet aj niekto iny